### PR TITLE
Initial v1.30 documentation

### PR DESCRIPTION
**Pull request by katcosgrove**
Adding initial documentation for v1.30 release and to discuss the potential release timeline.

### CODE REVIEW

File: releases/release-1.30/README.md
Comment (gracenng) [inline releases/release-1.30/README.md:89]: Date on this one needs an update :)

File: releases/release-1.30/README.md
Comment (gracenng) [inline releases/release-1.30/README.md:99]: fyi I didn't do this for the v1.28 release because it didn't feel necessary but Priyanka did

File: releases/release-1.30/README.md
Comment (xmudrii) [inline releases/release-1.30/README.md:59]: We tend to avoid Monday/Friday releases unless necessary because of availability of release managers and timezones. Would it work to move alpha.1 to Tuesday, Jan 30?

File: releases/release-1.30/README.md
Comment (xmudrii) [inline releases/release-1.30/README.md:69]: I have two concerns about this release:

- The time difference between alpha.2 and alpha.3 is almost one month. I'm wondering if we should have one release in between, e.g. on Feb 27
- Given our experience with alpha.3 in 1.29, I think we should consider moving this release for a few days after the code freeze
  - It's safe to expect that we'll have PRs merging on Thursday (the release day) and we'll also get a plenty of exception requests
  - All that increases chances to destabilize the CI signal, making it even more pressuring to both the CI signal team and the release managers

File: releases/release-1.30/README.md
Comment (xmudrii) [inline releases/release-1.30/README.md:81]: Maybe we could have beta.0 on March 5 or March 6 (closer to the beginning of the code freeze)

File: releases/release-1.30/README.md
Comment (xmudrii) [inline releases/release-1.30/README.md:77]: `release-1.30` branch and jobs are created upon cutting the rc.0 release. We should either move rc.0 to March 19 or move those two milestones to March 26

**Review by xmudrii — COMMENTED** [review]
Some nits related to the releases schedule, otherwise LGTM!

File: releases/release-1.30/README.md
Comment (katcosgrove) [reply releases/release-1.30/README.md:69]: I'm happy to have an alpha on Feb 27 so there's less of a gap, but moving alpha.3 (or alpha.4, if we add one on Feb 27) further ahead of code freeze (to March 12, the following Tuesday) will create a conflict with the suggestion of moving beta.0 to the beginning of code freeze.

What do you think about the following:

alpha.2 on Tuesday Feb 13
alpha.3 on Tuesday Feb 27
alpha.4 on Tuesday March 5 (beginning of code freeze)
beta.0 on Tuesday March 12 (week after code freeze)

This gets us an extra alpha, moves beta.0 out of kubecon week, and leaves a one week gap between beta.0 and rc.0 in case something gets screwy and we decide we need a beta.1 during kubecon week. 

I'm also fine moving alpha.3 to Feb 27, not adding an alpha.4, and moving beta.0 to either March 12 or March 5.

Comment (xmudrii) [reply releases/release-1.30/README.md:69]: > I'm also fine moving alpha.3 to Feb 27, not adding an alpha.4, and moving beta.0 to either March 12 or March 5.

+1, I think we don't need alpha.4, given that the CI signal might be shaky at that time and that we might not have the full capacity to deal with it (because of preparations for KubeCon), it seems safer/easier without it.

I'd say we could go with alpha.3 on Feb 27 and beta.0 on March 12, this seems like the safest approach.

File: releases/release-1.30/README.md
Comment (katcosgrove) [reply releases/release-1.30/README.md]: Yeah fair enough, it's redundant and it makes the doc super long. Removed

File: releases/release-1.30/README.md
Comment (xmudrii) [inline releases/release-1.30/README.md:72]:
```suggestion
| 1.30.0-beta.0 released                                                                 | Branch Manager                | Tuesday 12th March 2024                                                                                           | week 10  |                                                        |
```

**Review by xmudrii — COMMENTED** [review]
Just a nit, otherwise LGTM

**Review by xmudrii — APPROVED** [review]
/lgtm
/hold
for other folks to take a look

**Comment by katcosgrove** [comment]
cc: @kubernetes/sig-release-leads @kubernetes/release-managers

**Review by cpanato — APPROVED** [review]
/lgtm

thanks for doing this!

**Review by gracenng — APPROVED** [review]

**Pull request merged**
